tests/component-tests/pager/js/PagedTableTests.js

```diff
@@ -222,7 +269,7 @@ https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
                 options: {
                     moduleSource: {
                         func: "fluid.tests.tooltipModuleSource",
-                        args: "{pager}"
+                        args: ["{pager}", [2, 3, 4]]
```

Comment:
If we break it out into a variable it will be easier to know what it is just by reading the code, and it allows us to use it in other places if need be.